lib/ddlog_std.dl

```diff
@@ -209,11 +209,8 @@ extern function default(): 'T
 
 /* Returns the contained `Some` value or a provided default.
  */
-function unwrap_or(x: Option<'A>, def: 'A): 'A = {
-    match (x) {
-        Some{v} -> v,
-        None -> def
-    }
+function unwrap_or(#[by_val] x: Option<'A>, def: 'A): 'A = {
```

Comment:
are these used much?

Reply:
yes, especially in projects that declare many structs with `Option<>` fields.